clang/lib/AST/ByteCode/FixedPoint.h

```diff
@@ -91,6 +91,31 @@ class FixedPoint final {
     return ComparisonCategoryResult::Greater;
   }
 
+  size_t bytesToSerialize() const {
+    return sizeof(uint32_t) + (V.getValue().getBitWidth() / CHAR_BIT);
```

Comment:
I wonder how much do we care about `CHAR_BIT != 8`. Or you just avoid unnamed constants in the code?

Reply:
Yeah, _if_ I use it, it's just because of the unnamed constants.